be/src/exec/olap_scan_node.h

```diff
@@ -81,8 +80,8 @@ class OlapScanNode : public ScanNode {
     };
 
     typedef std::priority_queue<HeapType, std::vector<HeapType>, MergeComparison> Heap;
-
-    void display_heap(Heap& heap) {
+    
```

Comment:
my IDE has the same behavior as you for a blank line. I wonder should we keep non-space and non-tab style before a blank line? @morningman